pynsee/download/_load_data_from_schema.py

```diff
@@ -2,19 +2,22 @@
 import pandas as pd
 from functools import lru_cache
 import difflib
+import logging
 
 from pynsee.download._unzip_pb import _unzip_pb
 
+logger = logging.getLogger(__name__)
 
 @lru_cache(maxsize=None)
 def warning_file(missingFile, foundFile):
 
-    print(f"Data file missing in the zip file:\n{missingFile}")
+    msg = f"Data file missing in the zip file:\n{missingFile}\n"
     if not foundFile == "":
-        print(f"Following file has been used instead:\n{foundFile}")
+        msg += f"Following file has been used instead:\n{foundFile}\n"
     else:
-        print("No replacement file has been found")
-    print("Please report this issue")
+        msg += "No replacement file has been found\n"
+    msg += "Please report this issue !"
+    logger.error(msg)
```

Comment:
I'd say this is still an error if you tried to get a file and no file has been found (wether or not you found a replacement file which may be outdated). But in case we want to change the behaviour we could use a getattr. Something like this :
```
if blah:
  method = "warning"
else:
  method = "error"
getattr(logger, method)("Blah")
```

Reply:
cc @hadrilec for discussion (I have no strong opinion here)